modules/image-loading-optimization/assets/js/editor/component.js

```diff
@@ -1,8 +1,7 @@
 
-import ComponentBase from 'elementor-api/modules/component-base';
```

Comment:
I noticed that importing this made the bundled module file pretty large, while extending it using the global this way made the file around 20kb smaller. This pattern happens in a lot of places in Elementor, so adding some code splitting could be a nice benefit, but outside the scope of this PR for now.